app/assets/stylesheets/content/_tooltips.md

```diff
@@ -107,6 +107,29 @@ These can contain simple texts but are not suitable for HTML within the Tooltip.
 
 Note that the tabindex has to be set manually on the `<span>` and not the containing element. `tabindex="0"` makes the item tabbable at all.
 
+### HTML tooltips
+
+```
+<div class="form--field">
+  <label class="form--label" for="new_password" title="New password">New password</label>
```

Comment:
could you aim for a more general use case? i would like to be able to bind the tooltip to arbitrary elements in the future, not just that specific form)